templates/docker-compose/docker-compose.prod.yml.dist

```diff
@@ -41,6 +41,33 @@ services:
             - db_password
         command: 'bash -c "service cron start && tail -f /var/log/cron.log"'
 
+    supportpal_websockets:
+        container_name: '${WS_SERVICE_NAME}'
+        image: 'public.ecr.aws/supportpal/helpdesk:${APP_VERSION}-ws-buster'
+        restart: always
+        depends_on:
+            - db
+            - redis
+            - mailer
+        environment:
+            REDIS_HOST: ${CACHE_SERVICE_NAME}
+            REDIS_PORT: ${REDIS_PORT}
+            HOST: ${HOST}
+            MAILER_SERVICE_NAME: ${MAILER_SERVICE_NAME}
+            WS_SERVICE_NAME: ${WS_SERVICE_NAME}
+        volumes:
+            - supportpal_resources:/supportpal/resources
+            - supportpal_app:/supportpal/app
+            - supportpal_config:/supportpal/config/production
+            - supportpal_storage:/supportpal/storage
+            - ./customization/:/customization
+        networks:
+            - supportpal_backend
```

Comment:
It needs to be in the `supportpal_frontend` network to allow public connections on port 6001, or we need to change the nginx config so that it proxies web socket connections to this container - described at https://beyondco.de/docs/laravel-websockets/basic-usage/ssl

In this context (docker) it probably makes sense to proxy via the nginx config, then it's easier to manage the SSL config i.e. it's all handled by nginx rather than having to add SSL certs to the ws:serve server too.